src/main/java/net/sf/jabref/logic/formatter/bibtexfields/HtmlToUnicodeFormatter.java

```diff
@@ -0,0 +1,36 @@
+package net.sf.jabref.logic.formatter.bibtexfields;
+
+import net.sf.jabref.logic.formatter.Formatter;
+import net.sf.jabref.logic.l10n.Localization;
+import net.sf.jabref.logic.layout.LayoutFormatter;
+
+import org.apache.commons.lang3.StringEscapeUtils;
+
+public class HtmlToUnicodeFormatter implements LayoutFormatter, Formatter {
```

Comment:
Yes, but not a single class of the package `net.sf.jabref.logic.formatter.*` is listed in this section. I don't think a patchwork solution, where I document just one of those, would be a good idea. The whole page desperately needs updating.